examples/simple/src/comments/PostQuickCreate.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Create, LongTextInput, SimpleForm, TextInput } from 'react-admin'; // eslint-disable-line import/no-unresolved
+
+const PostQuickCreate = props => (
+    <Create {...props}>
+        <SimpleForm
+            defaultValue={{ average_note: 0 }}
+            validate={values => {
+                const errors = {};
+                ['title', 'teaser'].forEach(field => {
+                    if (!values[field]) {
+                        errors[field] = ['Required field'];
+                    }
+                });
+
+                if (values.average_note < 0 || values.average_note > 5) {
+                    errors.average_note = ['Should be between 0 and 5'];
+                }
+
+                return errors;
+            }}
+            redirect={false}
+        >
+            <TextInput source="title" />
+            <LongTextInput source="teaser" />
+        </SimpleForm>
+    </Create>
+);
+
+PostQuickCreate.defaultProps = {
+    basePath: '/posts',
+    formName: 'post-create',
```

Comment:
why do you set them as default props instead of passing them as props to `Create` in JSX? It would make the intent clearer.

Reply:
Less to write where you use it. But both are ok I guess

Reply:
I prefer in props than in default props

Reply:
Sure, no prb